# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/nsCoreUtils.h
# Commit: 09de143215a1
# Full Hash: 09de143215a13a22dcb2cd53d4ed1dfb60480c82
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-09-30 02:48 +0300
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Revert "Bug 1987945: apply code formatting via Lando" for causing bustages at DocAccessible.cpp.
# ==============================================================================

diff -r 826eb4ac00dd -r 09de143215a1 accessible/base/nsCoreUtils.h
--- a/accessible/base/nsCoreUtils.h	Tue Sep 30 02:35:22 2025 +0300
+++ b/accessible/base/nsCoreUtils.h	Tue Sep 30 02:48:50 2025 +0300
@@ -353,25 +353,6 @@
            aContent->IsGeneratedContentContainerForAfter() ||
            aContent->IsGeneratedContentContainerForMarker();
   }
-
-  /**
-   * Return the anchor frame for the given CSS positioned frame, or null if:
-   * 1. there is none,
-   * 2. there is more than one anchor,
-   * 3. or, there is one or more anchor used for sizing/margin only.
-   */
-  static nsIFrame* GetAnchorForPositionedFrame(
-      const PresShell* aPresShell, const nsIFrame* aPositionedFrame);
-
-  /**
-   * Return the CSS positioned frame for the given anchor frame, or null if:
-   * 1. there is none,
-   * 2. the anchor has more than one positioned frame,
-   * 3. or, there is one or more positioned frame using this anchor for
-   * sizing/margin only.
-   */
-  static nsIFrame* GetPositionedFrameForAnchor(const PresShell* aPresShell,
-                                               const nsIFrame* aAnchorFrame);
 };
 
 #endif